client/components/data/query-plugins/index.tsx

```diff
@@ -1,6 +1,11 @@
+import { useSelector } from 'react-redux';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import QueryAllJetpackSitesPlugins from '../query-all-jetpack-sites-plugins';
 import QueryJetpackPlugins from '../query-jetpack-plugins';
 
 export default function QueryPlugins( { siteId }: { siteId?: number } ) {
+	const isLoggedIn = useSelector( isUserLoggedIn );
+	if ( isLoggedIn === false ) return false;
```

Comment:
Any chance you could add some curly braces and line breaks here to stay with [WordPress JS](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#blocks-and-curly-braces) and [Calypso](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md#blocks-and-curly-braces) coding standards?

Reply:
Why isn't the linter/fixer picking this up?

Reply:
I think checking these `if` one-liners fell through the cracks when we started to use Prettier and then removed all formatting-related ESLint rules. One of them was the [`curly`](https://eslint.org/docs/latest/rules/curly) rule. We should reintroduce `curly` if we want to enforce this style (good idea IMO). It doesn't conflict with Prettier, which accepts and formats both forms (`IfStatement` + `ReturnStatement` AST vs `IfStatement` + `Block` + `ReturnStatement`), just adds on top of it.

Reply:
Thanks for noting that, @obenland. Fixed [it here](https://github.com/Automattic/wp-calypso/pull/68835/commits/74784889d9438c5d210fdaf8e85dbdf598ed76a1).

Reply:
Updated eslint here: #68923